Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -60,5 +60,7 @@
   "loc.messages.ExtensionListFailedError": "Failed to retrieve list of extensions installed in Azure App Service. Error %s.",
   "loc.messages.ExtensionListFailedResponseError": "Failed to retrieve list of extensions installed in Azure App Service. Status Code: %s (%s)",
   "loc.messages.InstallingSiteExtension": "Installing site extension '%s' in Azure App Service.",
-  "loc.messages.FailedToFetchAppServiceState": "Failed to fetch App Service State for App Service: %s, Error Code: %s, Error Message: %s"
+  "loc.messages.FailedToFetchAppServiceState": "Failed to fetch App Service State for App Service: %s, Error Code: %s, Error Message: %s",
+  "loc.messages.UnableToFetchAuthorityURL": "Unable to fetch authority url.",
+  "loc.messages.UnableToFetchActiveDirectory": "Unable to fetch active directory resource id."
```

Comment:
Is it possible to add reasons, why we are unable to fetch authority URL or resource ID?
In this way, user can debug the issue without ask.